### ISSUE TITLE
Allow launching item with 'wrapper' (optirun, primusrun, screen recording software)

### ISSUE THREAD

**Issue by fleskesvor** [itchio/itch#342]
As mentioned in issues #86 and #92, it's possible to create a launch script in the installation directory that will be executed instead of the game executable. It's not very user friendly though, so I wondered if it might be an idea to allow users to set launch options from the client, like you can in the Steam client, to enable users to add workarounds for possible issues, make it easier to use itch.io with screen recording software and to let users on Optimus laptops easily set a game to launch with their dedicated GPU.

**Comment by fasterthanlime** [comment]
Yep, definitely something I want to do.

For screen recording & optimus though, I'd like to add actual support for that in the app at some point.

Easy recording + upload of all itch games would be so cool :)

**Comment by fasterthanlime** [comment]
(This issue is blocked by #285 btw.)

**Comment by fasterthanlime** [comment]
(Sister issues: #366 / #357)

**Comment by fasterthanlime** [comment]
@fleskesvor Actually this is not the same issue as #366 — you don't want to (only) be able to pass command-line arguments to games, you also want to be able to launch them using a wrapper, a-la `optirun` or `primusrun`. Right?

**Comment by fleskesvor** [comment]
Yes, that is correct. I'd have to be able to trigger `optirun` or `primusrun` before the invocation that launches the game. The same goes for screen recording with e.g. [SimpleScreenRecorder](http://www.maartenbaert.be/simplescreenrecorder/) (not that I personally use it much) until that is supported by the client (which I agree would be very cool). In the Steam client you can do both, but if you want to wrap the launch command, which is set by the developer, including launch options, you have to type out e.g. `primusrun %command%` or `ssr-glinject %command%`.

**Comment by leafo** [comment]
There's now "launch options" which supports command wrapper mode, found in the Manage dialog for installed games.

<img width="966" height="249" alt="Image" src="https://github.com/user-attachments/assets/83e3fced-0a41-4255-950b-1e7fa5712db5" />
